.github/workflows/foreman.yml

```diff
@@ -33,7 +33,14 @@ jobs:
           POSTGRES_PASSWORD: password
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.setup_matrix.outputs.matrix) }}
+      matrix:
+        ruby: ${{ fromJson(needs.setup_matrix.outputs.matrix).ruby }}
+        node: ${{ fromJson(needs.setup_matrix.outputs.matrix).node }}
+        postgresql: ${{ fromJson(needs.setup_matrix.outputs.matrix).postgresql }}
+        task:
```

Comment:
I really wish GH would allow `include` to create a new axis…

Reply:
I wonder if `gha-matrix` should gain a `base-matrix` param, which is used a base and then `matrix.json` is joined therein?